src/viking.js

```diff
@@ -1,11 +1,98 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    }
+
+    // Iteration 1: Soldier
+attack() {
+    return this.strength;
+};
+
+receiveDamage(damage) {
+    this.damage = damage;
```

Comment:
here you're setting the damage as this.damage, but later you're just using damage, so you could have skipped that line